bin/bulk_load

```diff
@@ -7,15 +7,12 @@ LIBRARY_PATH = PROJECT_ROOT + "lib/"
 
 $LOAD_PATH << LIBRARY_PATH unless $LOAD_PATH.include?(LIBRARY_PATH)
 
-INDICES = %w{mainstream detailed government}.freeze
-
 require "rummager"
 
 Slop.parse(help: true) do
-  search_config = SearchConfig.new
-  banner %{Usage: #{File.basename(__FILE__)} {#{search_config.index_names.join("|")}}
+  banner %{Usage: #{File.basename(__FILE__)}
```

Comment:
Missing a closing brace?

Reply:
Moved to using <<~DOC..DOC as the multiline string with `%{..}` is rubbiush